backend/src/main/java/kr/touroot/TourootApplication.java

```diff
@@ -11,6 +12,8 @@
 public class TourootApplication {
 
     public static void main(String[] args) {
-        SpringApplication.run(TourootApplication.class, args);
+        new SpringApplicationBuilder()
+                .environment(new StandardEncryptableEnvironment())
+                .sources(TourootApplication.class).run(args);
```

Comment:
```suggestion
                .sources(TourootApplication.class)
                .run(args);
```

사소한 부분이지만 컨벤션을 지키기 위해 `.run()` 부분도 한 줄 내려주시죠!

Reply:
docs 의 코드를 그대로 복붙하다보니 컨벤션을 놓쳤네요 ㅎㅎ
감사합니닷!!